app/components/Counter.js

```diff
@@ -25,34 +25,42 @@ export default class Counter extends Component<Props> {
     } = this.props;
     return (
       <div>
-        <div className={styles.backButton} data-tid="backButton">
+        <div className={styles.backButton} data-tid={'backButton'}>
           <Link to={routes.HOME}>
-            <i className="fa fa-arrow-left fa-3x" />
+            <i className={'fa fa-arrow-left fa-3x'} />
           </Link>
         </div>
-        <div className={`counter ${styles.counter}`} data-tid="counter">
```

Comment:
Why are the quotes changed? Is there a particular reason for this change? If not its best to stick with convention

Reply:
I believe the rule is off by default, and one convention should be followed. I switched the rule from "always" to "never" now.